### PR TITLE
Loom clean up

### PR DESCRIPTION
**Pull request by alex-page**
Co-Authored-By: Ben Scott <227292+BPScott@users.noreply.github.com>

### CODE REVIEW

File: .changeset/moody-geckos-sparkle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+Removed jestAdjustmentsPlugin as we have the root jest test and moved scripts to scripts/build.js
```
Comment (alex-page) [inline .changeset/moody-geckos-sparkle.md]:
```suggestion
Removed jestAdjustmentsPlugin as we have the root jest config file. Moved scripts to scripts/build.js.
```

File: .changeset/moody-geckos-sparkle.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+Removed jestAdjustmentsPlugin as we have the root jest config file. Moved scripts to scripts/build.js.
```
Comment (alex-page) [inline .changeset/moody-geckos-sparkle.md]:
```suggestion
Removed jestAdjustmentsPlugin as we have `jest.config.js`. Moved scripts to scripts/build.js.
```

**Comment by BPScott** [comment]
We did this as a first step to removing loom as I wasn't sure how far we'd get. Turns out we got pretty dang far. I'm tempted to say we close this, and keep all the "delete loom" stuff in #5960

**Comment by alex-page** [comment]
I agree Ben! Going to merge it anyway and start hacking on the other PR.

**Pull request merged**